### PR TITLE
Reconnect terminated streams 

### PR DESCRIPTION
**Pull request by crajcan**

### CODE REVIEW

File: crates/http-source/src/main.rs [hunk 1]
```diff
@@ -13,29 +15,39 @@ use fluvio_connector_common::{
...
-        trace!(?item);
-        producer.send(RecordKey::NULL, item).await?;
+    let mut backoff = Backoff::new();
+
+    loop {
+        let mut stream = if config.stream {
+            match reconnect_stream_with_backoff(&config, &mut backoff).await {
+                Ok(stream) => stream,
+                Err(_err) => {
+                    continue;
```
Comment (digikata) [inline crates/http-source/src/main.rs:35]: Not sure the best place to put it, but there should be a way for a user to see and assess reconnects in the regular logs.  I think a `warn` or `error` log output on disconnect, and an `info` on successful connect would be sufficient for that.

**Review by digikata — CHANGES_REQUESTED** [review]
Looks good overall, some minor input on visibility of reconnects.

**Review by digikata — APPROVED** [review]
Nice, thanks for the update!

**Review by ajhunyady — COMMENTED** [review]
does this update require a patch number change?
how can I tell if it's in the build and ready to test?

**Comment by crajcan** [comment]
> does this update require a patch number change?
> 
> how can I tell if it's in the build and ready to test?

It has been merged but not released. I'm not quite sure what the procedure is, it looks like Alex has done the last few.

**Comment by ajhunyady** [comment]
@digikata we may need a `checklist` doc with instructions on how to release a connector.

**Comment by digikata** [comment]
For this connector the procedure to push a tag to the repository that is formatted `vX.Y.Z`

**Comment by digikata** [comment]
I added a release instruction template.  @crajcan Can you try it out and see if it makes sense to you?

**Pull request merged**